laborator/content/stiva/0-mean/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.o
```

Comment:
```suggestion
```
Do the same for all `.gitignore` files. `*.o` is already present in the repo's `.gitignore` [1]. No need to add it here as well. `.gitignore` files are additive. Each one is applied "down" the file hierarchy from where the file is placed. [1] is applied to the entire repo, for example, because it's placed at the root of the hierarchy.

[1] https://github.com/systems-cs-pub-ro/iocla/blob/0633936479a0277a59ce75ae40b7bab5f3424bdf/.gitignore#L5